loop2/FindOutMinAndMax.java

```diff
@@ -0,0 +1,19 @@
+package loop2;
+//Վերցնում եք ամբողջ թվերի զանգված։ Մի ցիկլով գտնում եք տվյալ զանգվածի միջի մինիմում ու մաքսիմում թվերը։
+public class FindOutMinAndMax {
+    public static void main(String[] args) {
+        int[] arr = {1, 1, 58, 1, 0, 1, -89};
+        int min = arr[0];
+        int max = arr[0];
+        for (int i = 1; i < (arr.length ); i++) {
+           if (arr[i] > max) {
+               max = arr[i];
```

Comment:
Is there any meaning to continue iteration if this condition is true? If no, what you're supposed to do?